acceptance/tests/puppet_device_test.rb

```diff
@@ -64,5 +66,5 @@
 on(master, "umask 077; puppet device #{common_options} --target spinny2.example.com", acceptable_exit_codes: [0, 1]) do |result|
   assert_no_match(/Permission denied/, result.stderr, 'cert requesting failed with --libdir')
 end
-on(master, puppet("ca sign spinny2.example.com"))
+on(master, "puppetserver cert sign spinny2.example.com --allow-dns-alt-names")
```

Comment:
`puppet cert` or `puppetserver ca`?

Reply:
I tried both: using `puppetserver ca` requires changes to the puppetserver presuites in 5.x to add `server` to the config. That's doable, I would just need to open a PR against that repo that would then block this. I went for the simpler option, but don't have a strong opinion.

In later branches this test is already updated to use `puppetserver ca`, but there the presuites support it.

Reply:
I mean, this line says `puppetserver cert` and I assume you mean one of `puppet cert` or `puppetserver ca`, but not actually `puppetserver cert`, right?

Reply:
`puppet cert` makes sense and matches the change higher up in this file.

Reply:
Oh derp